plugin-server/src/capabilities.ts

```diff
@@ -19,7 +19,7 @@ export function getPluginServerCapabilities(config: PluginsServerConfig): Plugin
                 processAsyncOnEventHandlers: true,
                 processAsyncWebhooksHandlers: true,
                 sessionRecordingBlobIngestion: true,
-                personOverrides: config.POE_DEFERRED_WRITES_ENABLED,
+                personOverrides: false, // not on by default, yet
```

Comment:
Going to need to remember to enable this once persons-on-events is ready for hobby.